public/app/features/alerting/unified/mocks/server/handlers/grafanaRuler.ts

```diff
@@ -28,6 +29,12 @@ export const rulerRulesHandler = () => {
   });
 };
 
+export const prometheusRulesHandler = () => {
+  return http.get('/api/prometheus/grafana/api/v1/rules', () => {
+    return HttpResponse.json<PromRulesResponse>({ status: 'success', data: { groups: [] } });
+  });
+};
```

Comment:
I didn't use the handler generic here because both the params and the request body are empty (producing something like `http.get<{}, {}, PromRulesResponse>`) which I've always found a bit weird and I can never remember which generic represents what 😅